apps/system/js/app_window_manager.js

```diff
@@ -626,7 +626,12 @@
       this._apps[evt.detail.instanceID] = evt.detail;
     },
 
-    '_handle_homescreen-changed': function() {
+    '_handle_homescreen-changed': function(evt) {
+      if (this.ftuLauncher &&
+          !this.ftuLauncher.respondToHierarchyEvent(evt)) {
```

Comment:
it's a bit weird since `homescreen-changed` isn't an event handled by the `HierarchyManager`.
can you call `ftuLauncher._handle-homescreen-changed` directly?